### ISSUE TITLE
does this plugin only work for neovim >v0.10 ? 

### ISSUE THREAD

**Issue by rvdwijngaard** [loganswartz/sunburn.nvim#1]
Hi, 

If I run this plugin with latest neovim (v0.9.5) I get the following error: 
```
E5113: Error while calling lua chunk: ...share/nvim/lazy/polychrome.nvim/lua/polychrome/group.lua:114: attempt to call field 'iter' (a nil value)
stack traceback:
	...share/nvim/lazy/polychrome.nvim/lua/polychrome/group.lua:114: in function <...share/nvim/lazy/polychrome.nvim/lua/polychrome/group.lua:109>
	...nvim/lazy/polychrome.nvim/lua/polychrome/colorscheme.lua:152: in function '_inject_gui_features'
	...nvim/lazy/polychrome.nvim/lua/polychrome/colorscheme.lua:52: in function 'define'
	...local/share/nvim/lazy/sunburn.nvim/lua/sunburn/theme.lua:5: in main chunk
	[C]: at 0x738cb64d0660
	[C]: in function 'require'
	...n/.local/share/nvim/lazy/sunburn.nvim/colors/sunburn.lua:8: in main chunk
```

With latest nightly build of nvim v0.10 everything works fine. Just wanted to confirm if this plugin is specifically designed for v0.10? If so, it might be helpful to mention that in the readme of this repo. Thanks for your hard work on this plugin!

**Comment by loganswartz** [comment]
Yes, currently `polychrome.nvim` (my other library I used to create this plugin) is Neovim 0.10+ only, although that wasn't necessarily intentional 😅

The reason for that error is because I'm trying to use `vim.iter`, which I didn't realize was quite so bleeding edge.

A quick and dirty fix for using this plugin in 0.9.5 and possibly even earlier versions is to use [this polyfill](https://github.com/mhanberg/iter.nvim) for `vim.iter`. If you change your config to look something like this (assuming you use `lazy.nvim` here), it should work:

```
{
    "loganswartz/sunburn.nvim",
    dependencies = {
        {
            "loganswartz/polychrome.nvim",
            dependencies = {
                {
                    "mhanberg/iter.nvim",
                    config = function()
                        vim.iter = require('iter')
                    end,
                },
            },
        },
    },
},
```

Ideally I would like to support more than just the bleeding edge version of Neovim, but I haven't gotten around to actually testing what the minimum working version is.

**Comment by rvdwijngaard** [comment]
Thanks for the clarification.
